kratos/python_scripts/analysis_stage.py

```diff
@@ -193,6 +193,49 @@ def ChangeMaterialProperties(self):
         """this function is where the user could change material parameters as a part of the solution step """
         pass
 
+    def ClearDatabase(self):
+        """ This method clears the database in case it is necessary
```

Comment:
what does it mean to "clear the database"?

Reply:
By default does nothing, but in the contact I sued in order to avoid conflicts (I remove the contact conditions and I create again later)

Reply:
In the @KratosMultiphysics/implementation-committee we have looked and it does not appear that you actually make use of 'ClearDatabase' in your implementations. The name is too vague, inviting developers to make dangerous implementations. Could elaborate on what actually you think this function should be able to do? What database are you thinking about?